src/dropkick.js

```diff
@@ -1103,6 +1103,7 @@ Dropkick.build = function( sel, idpre ) {
         case "OPTGROUP":
           optgroup = _.create( "li", { "class": "dk-optgroup" });
 
+
```

Comment:
![This guy](https://vignette1.wikia.nocookie.net/mlp/images/8/84/This-guy.jpg/revision/latest?cb=20130911170743)